portability-worker/src/main/java/org/dataportabilityproject/worker/JobMetadata.java

```diff
@@ -16,66 +16,67 @@
 package org.dataportabilityproject.worker;
 
 import com.google.common.base.Preconditions;
-import com.google.inject.Inject;
 import com.google.inject.Singleton;
 import java.security.KeyPair;
 import java.util.UUID;
 
 /**
- * A class that contains the value of the job and key for a worker instance. This classes uses the
- * initialization-on-demand holder idiom to ensure it is a singleton.
+ * A class that contains metadata for a worker's job.
+ *
+ * <p>This class is completely static to ensure it is a singleton within each worker instance.
  */
 @Singleton
```

Comment:
We can remove @Singleton because iff we're using the static singleton approach, this won't be guice injected

Reply:
Done